.circleci/config.yml

```diff
@@ -74,15 +74,15 @@ commands:
 
       - restore_cache:
           name: Restoring Go cache
-          key: go-mod-{{ arch }}-{{ checksum "go.sum" }}
+          key: go-mod-v1-{{ arch }}-{{ checksum "go.sum" }}
```

Comment:
What do these changes do?

Reply:
Because you can't delete caches in circle you have to update it - and in theory the checksum of go.sum should be enough to differentiate, but I was just forcing the issue in case there was something wrong with go.sum